rust/src/dns/log.rs

```diff
@@ -584,92 +573,142 @@ fn dns_log_json_answer(
         js_answers.close()?;
 
         if flags & LOG_FORMAT_DETAILED != 0 {
-            js.set_object("answers", &js_answers)?;
+            jb.set_object("answers", &js_answers)?;
```

Comment:
do we use `js_answers` if we do not have `LOG_FORMAT_DETAILED` ?

Reply:
No, this is up for further discussion tho.  Without detailed which pretty much only get the grouped results.

Reply:
Do you mean, it is not settled what should be in detailed mode ? and what should be in not detailed mode ?

Reply:
Yes. Non-detailed mode is currently nothing unfortunately. So if grouping is not enabled we default to detailed mode, otherwise the log is useless.